pkg/protocol/http1/req/request.go

```diff
@@ -289,19 +289,19 @@ func ContinueReadBodyStream(req *protocol.Request, zr network.Reader, maxBodySiz
 	if err != nil {
 		if errors.Is(err, errs.ErrBodyTooLarge) {
 			req.Header.SetContentLength(contentLength)
-			req.ConstructBodyStream(bodyBuf, ext.AcquireBodyStream(bodyBuf, zr, contentLength))
+			req.ConstructBodyStream(bodyBuf, AcquireRequestStream(bodyBuf, zr, contentLength, &req.Header))
```

Comment:
Why move ext.AcquireBodyStream here? Is There some different handle logic in request and response?

Reply:
we have to add header in stream to support read trailer,  `req.header` and  `resp.header` is different.